router/http_test.go

```diff
@@ -1005,6 +1004,7 @@ func (s *S) TestDefaultServerKeypair(c *C) {
 	assertGet(c, "https://"+l.TLSAddr, "foo.example.com", "2")
 }
 
+// TODO(bgentry): validate this in Postgres??
 func (s *S) TestCaseInsensitiveDomain(c *C) {
```

Comment:
I think this test is still useful b/c the domain in the request still has to be downcase'd.